src/platform/packages/shared/kbn-es-query/src/filters/helpers/compare_filters.ts

```diff
@@ -16,6 +16,7 @@ export interface FilterCompareOptions {
   index?: boolean;
   disabled?: boolean;
   negate?: boolean;
+  group?: boolean;
```

Comment:
`group` is used to specify whether the control belongs to a collapsible section or not - so we need to know when it changes just in case it goes from locally filtered to a specific section to global